src-plugins/itkFilters/itkMorphologicalFiltersToolBox.cpp

```diff
@@ -85,6 +85,7 @@ itkMorphologicalFiltersToolBox::itkMorphologicalFiltersToolBox ( QWidget *parent
     QLabel * morphoFilterLabel = new QLabel ( tr ( "Kernel radius:" ) );
     QHBoxLayout * morphoFilterLayout = new QHBoxLayout;
     d->mmButton = new QRadioButton(tr("mm"), this);
+    d->mmButton->setObjectName("mm");
```

Comment:
Actually I you remember, I didn't agree with you on that. I thought it would pose problems, for various reasons. For example, suppose you have a widget that contains two widgets, a label named "start" and a button with "start" written on it. How do you name each of those widgets ?
I think we should append the widget type to the name